bag_of_words/code/doc_labels.py

```diff
@@ -0,0 +1,36 @@
+# Extract the labels of documents, and store them into an array.

```

Comment:
Similar to the other files, I recommend writing functions and following PEP8 for this code.